docs/walkthrough.md

```diff
@@ -94,9 +94,31 @@ spec:
 
 </details>
 
+<summary>sample-app.service.yaml</summary>
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: sample-app
+  name: sample-app
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: sample-app
+  type: ClusterIP
+```
+
+</details>
```

Comment:
The opening `<detail>` block is missing above, hence the `sample-app.service.yaml` content isn't collapsible.

Reply:
good catch. fixed it :)